R/connect_inbo_dbase.R

```diff
@@ -136,7 +162,7 @@ on_connection_opened <- function(connection, code, type) {
         },
 
         # table preview code
-        previewObject = function(rowLimit, ...) {
+        previewObject = function(rowLimit, ...) { #nolint
```

Comment:
Only ignore the `object_name_linter()`
```suggestion
        previewObject <- function(rowLimit, ...) { #nolint: object_name_linter.
```

Reply:
@ThierryO Finally I changed `<-` back to `=`, as lintr interpreted the argument previewObject as a variable and gave the remark that this variable is defined but not used...